cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -133,6 +134,17 @@ public class Executor {
   private final KafkaCruiseControlConfig _config;
   private final AtomicDouble _partitionMovementCountPerSec;
   private final AtomicDouble _partitionMovementMbPerSec;
+  // The timers _proposalExecutionTimerInvolveBrokerRemoval and _proposalExecutionTimerInvolveBrokerDemotionOnly
+  // would be useful in estimating the cluster recover time during broker failures.
+  // Proposal execution for broker demotion would be faster compared to that for broker removal in general,
+  // so we have two metrics:
+  // (1) timer for broker removal (does not matter whether there is also broker demotion or not)
+  // (2) timer for broker demotion only
+  // Timer of executing proposal involved with broker removal (may or may not with broker demotion).
+  private final Timer _proposalExecutionTimerInvolveBrokerRemoval;
```

Comment:
> (may or may not with broker demotion).

To my understand, the removal process does not include any demotion. In that case, we can probably just name the var below as _proposalExecutionTimerInvolveBrokerDemotion;

Reply:
Yes, rebalances for broker removal tend to be pretty rare events from what I've seen, so a sticky gauge is probably fine.

Reply:
Sounds good.